docs/source/troubleshooting.md

```diff
@@ -138,3 +138,17 @@ but it failed with a "Kernel error" and a SSHException.**
 
     This should be done on the host running Enterprise Gateway as well as all the remote hosts
     on which the kernel is launched.
+
+- **I'm trying to use a notebook with user impersonation on a Kerberos authenticated
+cluster but it fails to authenticate.**
+
+    When using user impersonation in a YARN cluster with Kerberos authentication, if Kerberos is not
+    setup properly you will usually see the following warning that will keep a notebook from connecting:
+
+    ```bash
+    WARN Client: Exception encountered while connecting to the server : javax.security.sasl.SaslException: GSS initiate failed [Caused by GSSException: No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt)]
+    ```
+
+    The most common cause for this WARN is when the user that started Enterprise Gateway is not
+    authenticated with Kerberos. This can happen when the user has either not run `kinit` or their
+    previous ticket has expired.
```

Comment:
should we also add the command to properly init the user (e.g. if this was elyra user)

Reply:
The command is `kinit` so I'm not sure what to add, how a user authenticates is dependent on their server setup, `kinit` is just runs the default authentication for the user who runs it.